src/components/config/__tests__/create-config.test.ts

```diff
@@ -0,0 +1,64 @@
+/**********************************************************************************
+ * MIT License                                                                    *
+ *                                                                                *
+ * Copyright (c) 2021 Hyperjump Technology                                        *
+ *                                                                                *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy   *
+ * of this software and associated documentation files (the "Software"), to deal  *
+ * in the Software without restriction, including without limitation the rights   *
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      *
+ * copies of the Software, and to permit persons to whom the Software is          *
+ * furnished to do so, subject to the following conditions:                       *
+ *                                                                                *
+ * The above copyright notice and this permission notice shall be included in all *
+ * copies or substantial portions of the Software.                                *
+ *                                                                                *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     *
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       *
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    *
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         *
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  *
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  *
+ * SOFTWARE.                                                                      *
+ **********************************************************************************/
+
+import { expect } from 'chai'
+import fs from 'fs'
+import { createConfig } from '..'
+
+afterEach(() => {
+  if (fs.existsSync('monika.har.json')) {
+    fs.unlinkSync('monika.har.json')
+  }
+
+  if (fs.existsSync('monika.postman.json')) {
+    fs.unlinkSync('monika.postman.json')
+  }
+})
+
+describe('Har config', () => {
+  describe('Create config from har file', () => {
+    it('should create config from har file', async () => {
+      const flags = {
+        har: './src/components/config/__tests__/form_encoded.har',
+        output: 'monika.har.json',
+      }
+      await createConfig(flags)
+      expect(fs.lstatSync('monika.har.json').isFile()).to.be.true
```

Comment:
can we also add expectation to make sure that the generated monika.har.json is as expected? just compare the content of the generated json with the expected json.